src/documentation/wiki/breakout-deck.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Same story as for the prototype deck: https://www.bitcraze.io/products/breakout-deck/